app/controllers/calendar_controller.rb

```diff
@@ -23,7 +23,8 @@ def create_calendar_props
               days: calendar_days,
               times: calendar_times,
               unitEventLengthInSeconds: EventGrouper::GRANULARITY,
-              date: params[:date]}.compact
+              date: params[:date],
+              scrollTo: {hours: 6, minutes: 0}}.compact
```

Comment:
Maybe it's better not to use controller in order to generate props for view logic